### ISSUE TITLE
Aerospike Server Enterprise Edition (Authentication Issue)

### ISSUE THREAD

**Issue by cqeescalona** [aerospike/aerospike-client-php#41]
Hi,

I'm trying to connect to an Aerospike Server Enterprise Edition version 3.5.8 with the PHP client but I can't. Actually I was developing on a community server with no problem at all, but once I upgraded to enterprise I got the issue.

This is basically what I'm doing:

 ```php
$config = [
    "hosts" => [
        ["addr" => "127.0.0.1", "port" => 3000]
    ],
    "user" => "someuser",
    "pass" => "somepassword"
];

$db = new Aerospike($config);

if (!$db->isConnected()) {
  echo "Failed to connect to the Aerospike server [{$db->errorno()}]: {$db->error()}\n";
  exit(1);
}

$result = array();
$callback = function ($record) use(&$result) {
    $result = $record['bins'];
};

$where = \Aerospike::predicateEquals("name", "test");
$status = $this->db->query('namespacetest', 'settest', $where, $callback);

if ($status === \Aerospike::OK) {
   return $result;
}
return null;

```

Actually it does connect to the server. 

The problem is when does the query. I receive the following error code:

**errorno = 80**
**error = AEROSPIKE_NOT_AUTHENTICATED** (User must be authentication before performing database operations.)

I've already created the user at the console using AQL commands. I granted permissions to this user. And I actually can read and write with this user (over the console only), but when try to use the client, just can't.

Thanks.

**Comment by rbotzer** [comment]
Hi Eidher. The security methods are not implemented yet for the PHP client, but are due soon.

**Comment by rbotzer** [comment]
We will do authentication separately from the security methods, and they'll be available soon.

**Comment by cqeescalona** [comment]
That's great Ronen. We'll be waiting for it. Thanks!

**Comment by rbotzer** [comment]
Fixed in the new release 3.3.14.

**Comment by cqeescalona** [comment]
Excellent! Thanks
